### PR TITLE
fix overflowed input

### PR DESCRIPTION
**Pull request by thomscoder**

### CODE REVIEW

**Comment by thomscoder** [comment]
There was an overflowed input in the releases tables. I think it was caused by some extra padding. This should fix the issue

**Comment by Trott** [comment]
Seems like the package-lock.json change is unrelated? Can we remove it?

**Comment by Trott** [comment]
I'm concerned that if/when we update the code from source, the bug will be re-introduced. I'm not sure what the best path here is. 

* Float this patch anyway?
* Put the unminified source in the repo here and minify it as part of our pipeline?
* Remove the dependency and replace it with something else or write it ourselves?
* Submit a fix upstream?
* Fix outside of the minified file via CSS or something else?

Part of the issue is...uh...I haven't looked closely at the source of the problem yet.

File: package-lock.json [hunk 1]
```diff
@@ -4,7 +4,6 @@
   "requires": true,
   "packages": {
     "": {
-      "name": "nodejs.org",
```
Comment (nschonni) [inline package-lock.json:7]: This change doesn't seem related

**Comment by thomscoder** [comment]
> Seems like the package-lock.json change is unrelated? Can we remove it?

Yes. My bad!

File: package-lock.json (hunk 1 above)
Comment (thomscoder) [reply package-lock.json:7]: Yes, my bad. Should've noticed the double push

**Comment by humble-barnacle001** [comment]
> There was an overflowed input in the releases tables. I think it was caused by some extra padding. This should fix the issue

Could please post a link and/or SS of the page so that I could look closely into the source?

**Comment by thomscoder** [comment]
> > There was an overflowed input in the releases tables. I think it was caused by some extra padding. This should fix the issue
> 
> Could please post a link and/or SS of the page so that I could look closely into the source?

Issue link: https://github.com/nodejs/nodejs.org/issues/4235
Bug's URL link: https://nodejs.org/en/download/releases/ (input in the table is overflowing)

**Comment by humble-barnacle001** [comment]
> I'm concerned that if/when we update the code from source, the bug will be re-introduced. I'm not sure what the best path here is.
> 
>     * Float this patch anyway?
> 
>     * Put the unminified source in the repo here and minify it as part of our pipeline?
> 
>     * Remove the dependency and replace it with something else or write it ourselves?
> 
>     * Submit a fix upstream?
> 
>     * Fix outside of the minified file via CSS or something else?
> 
> 
> Part of the issue is...uh...I haven't looked closely at the source of the problem yet.


I think what we could probably do is:
- Submit a fix to the upstream; If the person is unresponsive, we could then copy it. Till then I could probably patch the page with the `inputStyle` option of the dependency
- Use `jquery.fancytable` from npm instead of copying the file directly so that dependabot takes care of the update

What say @thomscoder ?

**Comment by humble-barnacle001** [comment]
Changing `static/js/previousVersion.js` to this (adding: `inputStyle: 'box-sizing: border-box;'`)
Screenshot from 2021-11-13 20-28-51

Fixes the issue without changing the source-code for the js because I think this is due to the CSS of the website rather than a bug.
I would those suggest to at least include the original source https://github.com/myspace-nu/jquery.fancyTable in the `static/js/fancyTable.min.js` to give credit as well as provide an uplink for reference. Also we could use the CDN link instead of copying it seperately.
This should probably be a new PR @thomscoder

**Comment by humble-barnacle001** [comment]
Alternatively, adding this CSS option (inspired by Bootstrap since the theme closely follows its pattern) in the file `layouts/css/_base.scss` is a worldwide fix for any such issue but needs to investigated in other pages to see possible disruptions in other pages

```Css
*,
*::before,
*::after {
  box-sizing: border-box;
}
```

**Comment by nschonni** [comment]
Opened up https://github.com/nodejs/nodejs.org/pull/4250 that might conflict with this, unless you get the change accepted upstream

**Comment by thomscoder** [comment]
> https://github.com/myspace-nu/jquery.fancyTable

Alright! Should I proceed with the fix? and we treat the conflict (@nschonni mentioned) that may arise separately?

**Comment by nschonni** [comment]
Looks like it's now been reported there too https://github.com/myspace-nu/jquery.fancyTable/issues/21

**Comment by humble-barnacle001** [comment]
> > https://github.com/myspace-nu/jquery.fancyTable
> 
> Alright! Should I proceed with the fix? and we treat the conflict (@nschonni mentioned) that may arise separately?

I think you could proceed with the fix in https://github.com/nodejs/nodejs.org/pull/4243#issuecomment-968082507 in a new PR after #4250 is patched through

**Comment by humble-barnacle001** [comment]
Fixed in https://github.com/myspace-nu/jquery.fancyTable/issues/21#issuecomment-968996194 version 1.0.25

**Comment by nschonni** [comment]
I've updated the #4250 with the new patched version

**Comment by nschonni** [comment]
Closing this one since I landed the updated version from NPM
